src/modules/func.js

```diff
@@ -0,0 +1,36 @@
+import postScore from "./post.js";
+import getData from "./get.js";
+
+const display = document.querySelector('.display');
+export default class UI{
+    static async getScorefromAPI () {
+        try {
+            const scores = await getData();
+            return scores;
+        } catch (error) {
+            throw error;
+        }
+    }
+    static async addScore  (user, score) {
+        try {
+            const addedScore = await postScore(user, score);
+            return addedScore;
+        } catch (error) {
+            throw error;
+        }
+    }
+    static libraryScores  (element) {
+        let nameScore = document.createElement('div');
+        nameScore.classList.add('even');
+        nameScore.innerHTML = `<div class='inputs'> ${element.user} : ${element.score}</div>`;
+        display.appendChild(nameScore);
+    }
+    static async displayScores () {
+        try {
+            const scores = await UI.getScorefromAPI();
+            scores.forEach((element) => UI.libraryScores(element));
```

Comment:
- [ ] There seems to be some bug in your app. When I click the "Refresh" button nothing is displayed from the API. Kindly fix that. The scores on the API should be fetched and displayed on the page when the button is clicked.

- [ ] When I add a score it updates the HTML page with `undefined` as the name instead of the actual name (as you can see I added Julie but it is listed as `undefined`). Kindly fix these errors.

  ![ruw](https://user-images.githubusercontent.com/81468675/216547295-6be9c20d-9735-4ee5-90cb-3ec421a47ad8.jpg)